release/microbenchmark/wait_cluster.py

```diff
@@ -0,0 +1,54 @@
+import argparse
```

Comment:
why do we need this file?

Reply:
I have no idea why it is created... I feel like the run_release tests automatically create this. Let me remove it.